.github/workflows/main-ci-cd.yml

```
@@ -3,7 +3,7 @@ name: CI/CD with GitHub Actions, Docker, and ECR
 
 on:
   push:
-    branches: [ "develop" ]
+    branches: [ "main" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)